libr/core/cmd_print.c

```diff
@@ -5368,6 +5368,12 @@ static int cmd_print(void *data, const char *input) {
 			r_core_cmd_help (core, help_msg_pd);
 			pd_result = 0;
 		}
+		if (formatted_json) {
```

Comment:
why this all is_html/was_html thing? It seems to me the changes at https://github.com/radareorg/radare2/pull/17628/files#diff-7b45b46dd8a84a924b91f34c2b1e6309R1208 already fixes the problem. What am I missing?

Reply:
The reason is because the ooutput json doesnt needs to be htmllized, only the disasm stuff before being esacped into the json. the was:html is used after flush to reset back the value in case a json output was printed. Note that the html output is VERY inneffiient right now i plan to optimize it in another pr

Reply:
Could you provide an example of what would happen without this? I'm not sure I understood

Reply:
maybe we should have an api in rcons to discard htmlization before flush even iff set instead of just setting a new value

Reply:
This is the diff with and without washtml:

```
-    "text": "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;font&nbsp;color='#c19c00'&gt;0x10000159a&lt;/font&gt;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;font&nbsp;color='#d0d0d0'&gt;lea&nbsp;&nbsp;&lt;/font&gt;&lt;font&nbsp;color='#f0f0f0'&gt;&nbsp;rdi&lt;/font&gt;,&lt;font&nbsp;color='#f0f0f0'&gt;&nbsp;&lt;/font&gt;&lt;font&nbsp;color='#f0f0f0'&gt;&nbsp;&lt;/font&gt;[&lt;font&nbsp;color='#c19c00'&gt;0x100004af5&lt;/font&gt;]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;;&nbsp;\"AB\""
+    "text": "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<font color='#c19c00'>0x10000159a</font>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<font color='#d0d0d0'>lea&nbsp;&nbsp;</font><font color='#f0f0f0'>&nbsp;rdi</font>,<font color='#f0f0f0'>&nbsp;</font><font color='#f0f0f0'>&nbsp;</font>[<font color='#c19c00'>0x100004af5</font>]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;;&nbsp;\"AB\""
```


note the spaces inside the tags are also nbsp'd which is wrong because it runs html twice